plugins/pci-psp/fu-pci-psp-device.c

```diff
@@ -390,5 +430,6 @@ fu_pci_psp_device_class_init(FuPciPspDeviceClass *klass)
 {
 	FuDeviceClass *device_class = FU_DEVICE_CLASS(klass);
 	device_class->probe = fu_pci_psp_device_probe;
+	device_class->setup = fu_pci_psp_device_setup;
```

Comment:
i think technically this should be an action in ->probe() -- it doesn't need ->open() and ->close() afterall.

Reply:
but we don't record events from probe, only from setup I thought.

Reply:
probe should work too

Reply:
Hmm, not sure why it's not catching events then.

Reply:
Can you pull these, and re-record your emulation pls.